docs/api/cypress-api/require.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: Cypress.require
+e2eSpecific: true
+---
+
+`Cypress.require` enables utilizing dependencies within the
+[`cy.origin()`](/api/commands/origin) callback function. It is used to require
+modules such as [npm](https://www.npmjs.com/) packages and other local files.
+
+It is functionally the same as using
+[CommonJS `require()`](https://nodejs.org/en/knowledge/getting-started/what-is-require/)
+in browser-targeted code.
+
+## Syntax
+
+```js
+Cypress.require(moduleId)
+```
+
+## Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```js
+cy.origin('example.com', () => {
+  const _ = Cypress.require('lodash')
+  const utils = Cypress.require('./utils')
+
+  // ... use lodash and utils ...
+})
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```js
+// `Cypress.require()` cannot be used outside the `cy.origin()` callback.
+// Use `require()` instead
```

Comment:
```suggestion
// Use CommonJS `require()` instead
```

Reply:
Addressed in https://github.com/cypress-io/cypress-documentation/pull/5020/commits/257a803e312e274f0e654dbe73ac7cc883a88d21